sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/socket.scala

```diff
@@ -128,8 +128,9 @@ class TextSocketSource(host: String, port: Int, includeTimestamp: Boolean, sqlCo
       batches.slice(sliceStart, sliceEnd)
     }
 
-    val rdd = sqlContext.sparkContext.parallelize(rawList).map(
-        v => InternalRow(UTF8String.fromString(v._1), v._2.getTime()))
+    val rdd = sqlContext.sparkContext.
```

Comment:
we generally put the '.' in the next line. So
```
val rdd = sqlContext.sparkContext
   .parall ...
   .map ....
```

Reply:
done